addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -181,24 +181,36 @@ def _search_virtual_remaining_leaves(self, operator, value):
 
         return [('id', 'in', valid_leave_types.ids)]
 
-    def get_days(self, employee_id):
-        # need to use `dict` constructor to create a dict per id
-        result = dict((id, dict(max_leaves=0, leaves_taken=0, remaining_leaves=0, virtual_remaining_leaves=0)) for id in self.ids)
+    def get_days(self, employee_ids):
+        old_api = False
```

Comment:
So v8! How about creating a new method `get_employees_days` and keeping `get_days` for backward compatibility
```py
# TODO remove me 14.0
return self.get_employees_days([employee_id])[0]
```

Reply:
I'd even suggest doing that for all the methods with backwards compatibility cases e.g. 64374ea83c0d7b2745c8e440826591b3d6630bc3

Reply:
Yep. I knew it was ugly :)

Thanks for the suggestion, that's much better indeed.